r/R/dplyr-funcs-datetime.R

```diff
@@ -615,4 +618,44 @@ register_bindings_datetime_parsers <- function() {
 
     build_expr("assume_timezone", coalesce_output, options = list(timezone = tz))
   })
+
+}
+
+
+register_bindings_datetime_rounding <- function() {
+
+  register_binding("round_date", function(x, unit = "second",
+                                          week_start = getOption("lubridate.week.start", 7)) {
```

Comment:
Very minor style nitpick, but should we have `unit = "second"` on a separate line in this form?

Reply:
https://style.tidyverse.org/functions.html#long-lines-1